docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -157,6 +157,33 @@ const sel = Selector('div').child();
 </html>
 ```
 
+The selector's value evaluates when you call this selector, not when you declare it. If the page's contents change since you've declared the selector, it can point to another element or none.
```

Comment:
Do you mean:

`The selector value is evaluated when you call the selector, not when you declare it. If the page content changes because you declared the selector, it may point to another element, or no element.`

Reply:
No, changes on the page are unrelated to the selector declaration. 

The selector value is evaluated when you call this selector, not when you declare it. If the page content changes since you've declared the selector **(by the time you call it)**, it can point to another element or no element.

Declare selector -> page **changes** -> use selector

said **changes** might influence selector value but are not caused by it.

If my version is misleading or unclear, please advise

Reply:
Okay. Is it technically correct to say:

`The selector value is evaluated when you call this selector, not when you declare it. If the page content changes after you have declared the selector, it may point to another element, or no element.`

Reply:
Yes, edited